NEWS.md

```
@@ -27,6 +27,17 @@
   * ``partial`` option is removed.
   * Function promoted to official.
 
+**Official functions changes**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading
null

(MD036, no-emphasis-as-heading)